Tests/Common/Scheduling/DateRulesTests.cs

```diff
@@ -313,6 +313,239 @@ public void EndOfMonthWithSymbolWithOffset(Symbols.SymbolsKey symbolKey, int[] e
             }
         }
 
+        [Test]
+        public void StartOfYearNoSymbol()
+        {
+            var rules = GetDateRules();
+            var rule = rules.YearStart();
+            var dates = rule.GetDates(new DateTime(2000, 01, 01), new DateTime(2010, 12, 31));
+
+            int count = 0;
+            foreach (var date in dates)
+            {
+                count++;
+                Assert.IsTrue(date.Day <= 4);
```

Comment:
Should this be `Assert.AreEqual(1, date.Day);`? 
Also, you could assert the month as well in all of these tests